code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1232,6 +1232,51 @@ GLOBAL_LIST_EMPTY(PDAs)
 			continue
 		. += P
 
+//borg pda stuff
+/mob/living/silicon/robot/proc/cmd_send_pdamesg(mob/user)
```

Comment:
I'd recommend, if possible, to straight up refractor the AI pda function so it's available for all silicons as it was on Skyrat